docs/content/ecs/entity-contracts.md

```diff
@@ -23,7 +23,7 @@ A reactive component is implemented as a temporary component, and so has the sam
 
 ## Components on ECS Entities that represent SpatialOS entities
 
```

Comment:
I am not going to comment on the changes not having "worker instance" as it's in all the docs without "instance" but, going forward, if we can say "worker instance" everywhere it's relevant that helps understanding.

And "has authority over" rather than "is authoritative over" also helps understanding, so we always use the same phrase.